packages/localize/src/tools/src/translate/source_files/source_file_utils.ts

```diff
@@ -89,6 +89,25 @@ export function unwrapMessagePartsFromLocalizeCall(call: NodePath<t.CallExpressi
         throw new BabelParseError(
             cooked.node, 'Unexpected "makeTemplateObject()" function (expected an expression).');
       }
+    } else if (right.isSequenceExpression()) {
+      const expressions = right.get('expressions');
+      if (expressions.length > 2) {
+        // This is a minified sequence expression, where the first two expressions in the sequence
+        // are assignments of the cooked and raw arrays respectively.
+        const first = expressions[0];
```

Comment:
What do you think of this?

```suggestion
        const [first, second] = expressions;
```

Reply:
I like it!